blog/views.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib import messages
+from django.contrib.auth.decorators import login_required
+from django.core.paginator import Paginator
+from django.shortcuts import render, redirect
+from django.views import generic
+
+from blog.models import Post, Commentary
+
+
+def index(request):
+    posts = Post.objects.all().order_by("created_time")
```

Comment:
Consider adding a filter to only show published posts, unless you want to show all posts regardless of their published status.